README.md

```diff
@@ -192,6 +192,7 @@ Collects following information from Google Chrome web browser:
  - History
  - Archived History
  - Cookies
+ - Extension
```

Comment:
It isn't under the firefox one... thought it might be intentional

On Mon, Jun 22, 2015 at 10:22 AM, Jakub Sendor <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Yelp/osxcollector/pull/95#discussion_r32959324>:
>
> > @@ -192,6 +192,7 @@ Collects following information from Google Chrome web browser:
> >   - History
> >   - Archived History
> >   - Cookies
> > + - Extension
>
> Why not plural?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Yelp/osxcollector/pull/95/files#r32959324>.
>